include/fluent-bit/flb_sha512.h

```diff
@@ -25,13 +45,26 @@
 
 #include <stdint.h>
 
+#ifdef FLB_HAVE_OPENSSL
```

Comment:
You should include `fluent-bit/flb_info.h` before using this constant, that's a configure time generated header that defines these macros.